docs/tutorial/notebook/train_ss_model_using_pytorch.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>In this tutorial, we will learn how to train a semantic segmentation model using PyTorch<s> in a Jupyter Notebook. We assume that you are familiar with Jupyter Notebook and have created a folder notebooks in a folder that is relative to ml3d. </s></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done.</p>